src/components/icons/Icon.tsx

```diff
@@ -200,13 +203,16 @@ export const IOIcons = {
   option: IconOption,
   key: IconKey,
   docGiacenza: IconGiacenza,
+  docPaymentCode: IconDocPaymentCode,
   docAttach: IconDocumentAttachment,
   docAttachPDF: IconDocumentAttachmentPDF,
+  notes: IconNotes,
   attachment: IconAttachment,
   add: IconAdd,
   success: IconSuccess,
   ok: IconOk,
   fiscalCodeIndividual: IconFiscalCodeIndividual,
+  iconEntityCode: IconEntityCode,
```

Comment:
Drop the `icon`. Just `entityCode`. It's cleaner (semicit.) 
```suggestion
  entityCode: IconEntityCode,
```

Reply:
Addressed into 519127df5db30d63d58db861f91991598ac738aa